src/components/OrdersForm.vue

```diff
@@ -90,32 +88,31 @@ export default {
   methods: {
     submitOrderForm () {
       if (this.edit === false) {
-        this.$store.dispatch('orderFormSubmit', {
+        this.$store.dispatch('orders/orderFormSubmit', {
           order: this.order,
           description: this.description,
           price: this.price
         })
         this.close()
       } else {
-        this.$store.dispatch('submitEditForm', {
+        this.$store.dispatch('orders/submitEditForm', {
           order: this.order,
           description: this.description,
           price: this.price,
           index: this.index
```

Comment:
What is this index then ? Try to compare it with what you have done previously and do it accordingly, you require that index for editing right?